drivers/serial/uart_pl011.c

```diff
@@ -506,7 +524,7 @@ static int pl011_init(const struct device *dev)
 	if (!data->sbsa) {
 		get_uart(dev)->dmacr = 0U;
 		barrier_isync_fence_full();
-		get_uart(dev)->cr &= ~(BIT(14) | BIT(15) | BIT(1));
+		get_uart(dev)->cr &= ~BIT(1);
```

Comment:
It is better to use `PL011_CR_SIREN` instead of `BIT(1)`.

Reply:
Changed as suggested.